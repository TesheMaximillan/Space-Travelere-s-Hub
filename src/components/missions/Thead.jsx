import React from 'react';

function Thead() {
  return (
    <tr>
      <th>Mission</th>
      <th>Description</th>
      <th>Status</th>
      <th> </th>
    </tr>
  );
}

export default Thead;
