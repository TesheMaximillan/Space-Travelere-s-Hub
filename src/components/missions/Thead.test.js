import React from 'react';
import renderer from 'react-test-renderer';
import '@testing-library/jest-dom';
import Thead from './Thead';
import renderContext from '../storeMock';

describe('Myprofile component', () => {
  let component;

  beforeEach(() => {
    component = renderer.create(renderContext(<Thead />));
  });

  it('should render with given state from redux store', () => {
    // Assert
    expect(component.toJSON()).toMatchSnapshot();
  });
});
