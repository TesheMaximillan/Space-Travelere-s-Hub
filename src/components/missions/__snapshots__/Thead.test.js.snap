// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Myprofile component should render with given state from redux store 1`] = `
<tr>
  <th>
    Mission
  </th>
  <th>
    Description
  </th>
  <th>
    Status
  </th>
  <th>
     
  </th>
</tr>
`;
